src/Microsoft.EntityFramework.SqlServer/readme.md

```diff
@@ -0,0 +1,138 @@
+- Suports new SQL Server features, including support for the SQL Server 2022 enchanced client protocol (TDS8)
+- Supports most Azure Active Directory authentication methods
+- Supports Always Encrypted with .NET
+
+Notice that this provider is a runtime only update, and will not work with the existing Visual Studio tooling.
+
+The latest build of this package is available from [NuGet](https://www.nuget.org/packages/Microsoft.EntityFramework.SqlServer)
+
+## Configuration
+
+There are various ways to configure Entity Framework to use this provider.
+
+You can register the provider in code using an attribute:
+
+````csharp
+[DbConfigurationType(typeof(MicrosoftSqlDbConfiguration))]
+public class SchoolContext : DbContext
+{
+    public SchoolContext() : base()
+    {
+    }
+
+    public DbSet<Student> Students { get; set; }
+}
+````
+If you have multiple classes inheriting from DbContext in your solution, add the DbConfigurationType attribute to all of them.
+
+Or you can use the SetConfiguration method before any data access calls:
+````csharp
+ DbConfiguration.SetConfiguration(new MicrosoftSqlDbConfiguration());
+````
+Or you can add the following lines to your existing DbConfiguration class:
+````csharp
+SetProviderFactory(MicrosoftSqlProviderServices.ProviderInvariantName, Microsoft.Data.SqlClient.SqlClientFactory.Instance);
+SetProviderServices(MicrosoftSqlProviderServices.ProviderInvariantName, MicrosoftSqlProviderServices.Instance);
+// Optional
+SetExecutionStrategy(MicrosoftSqlProviderServices.ProviderInvariantName, () => new MicrosoftSqlAzureExecutionStrategy());
+````
+You can also use App.Config based configuration:
+
+````xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <configSections>
+        <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />    
+    </configSections>
+    <entityFramework>
+        <providers>		
+            <provider invariantName="Microsoft.Data.SqlClient" type="System.Data.Entity.SqlServer.MicrosoftSqlProviderServices, Microsoft.EntityFramework.SqlServer" />
+        </providers>
+    </entityFramework>
+    <system.data>
+        <DbProviderFactories>
+           <add name="SqlClient Data Provider"
+             invariant="Microsoft.Data.SqlClient"
+             description=".NET Framework Data Provider for SqlServer"
+             type="Microsoft.Data.SqlClient.SqlClientFactory, Microsoft.Data.SqlClient" />
+        </DbProviderFactories>
+    </system.data>
+</configuration>
+````
+If you use App.Config with a .NET 6 or later app, you must register the DbProviderFactory in code once:
+
+````csharp
+DbProviderFactories.RegisterFactory(MicrosoftSqlProviderServices.ProviderInvariantName, Microsoft.Data.SqlClient.SqlClientFactory.Instance);
+````
```

Comment:
Why? Was it just for the Migraitons commands? If so, that should be fixed by #1919

Reply:
It was not migrations, it was during runtime. But I am unable to find the related issue. It can possibly be working anyway.

Reply:
It was due to this: https://github.com/dotnet/runtime/issues/31492 - so the line of code replaces the invalid "system.data" section.